SETUP_GUIDELINES.md

```diff
@@ -0,0 +1,34 @@
+## Steps for Developers 
+1. Download the latest stable release of [Android Studio](https://developer.android.com/studio/index.html).
+2. Fork the [Project Repository](https://github.com/openfoodfacts/openfoodfacts-androidapp) by clicking "Fork".
+3. Clone the repository `git clone https://github.com/YOUR_USERNAME/openfoodfacts-androidapp.git`.
+4. Initialize Git `git init`.
```

Comment:
Cloning automatically creates the .git  repository inside the repo  you don't need to explicitly  create it. 
You can put here `cd openfoodfacts-androidapp`

Reply:
Yes . I will make the change . thanks @sachincool

Reply:
@sachincool  Thanks ! I will make the required change .